src/components/user/repo/account.repo.js

```diff
@@ -13,8 +13,7 @@ const getUserAccount = async (userId, app) => {
 	return account;
 };
 
-const deleteUserAccount = async (userId, app) => {
-	const account = await getUserAccount(userId, app);
+const deleteUserAccount = async (account, app) => {
```

Comment:
please do not pass app in repo and uc layer

Reply:
done